packages/sdk/src/CLRunner.ts

```diff
@@ -1835,26 +1924,24 @@ export class CLRunner {
      * Replay a TrainDialog, calling EntityDetection callback and API Logic,
      * recalculating FilledEntities along the way
      */
-    public async ReplayTrainDialogLogic(trainDialog: CLM.TrainDialog, state: CLState, cleanse: boolean): Promise<CLM.TrainDialog> {
+    public async ReplayTrainDialogLogic(trainDialog: CLM.TrainDialog, state: CLState, cleans: boolean): Promise<CLM.TrainDialog> {
```

Comment:
Curious why the change from "cleanse" to "cleans"?

Reply:
Hmm looked like another typo. Didn't realize `cleanse` was spelled that way. Anyways, both the words are meaningless so it doesn't matter, will change it back.